website/projects/map4sci/src/app/app.component.ts

```diff
@@ -1,10 +1,90 @@
-import { Component } from '@angular/core';
+import { Any } from '@angular-ru/common/typings';
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { FeatureCollection } from 'geojson';
 
+const EMPTY_FEATURES: FeatureCollection = {
```

Comment:
Move all these interface/definitions to appropriate places in the map directory/component